networkx/convert.py

```diff
@@ -489,3 +507,24 @@ def from_edgelist(edgelist, create_using=None):
     G = nx.empty_graph(0, create_using)
     G.add_edges_from(edgelist)
     return G
+
+
+if TYPE_CHECKING:
+    from networkx.classes.graph import Graph, Node, EdgePlus
+
+    G = TypeVar("G", bound=Graph)
+
+from contextlib import suppress
```

Comment:
I believe this should be indented

Reply:
yes, the pull that was suggested was erroneous. I'll need to go over the whole PR again before submitting regardless.

Reply:
no worries, fantastic effort on this!

Reply:
mea culpa

Reply:
@danieleades no worries, can you fix it?